packages/suite/src/components/firmware/FirmwareProgressBar.tsx

```diff
@@ -1,54 +1,15 @@
 import styled, { useTheme } from 'styled-components';
 
-import { Icon, ProgressBar, variables } from '@trezor/components';
-import { borders, spacingsPx } from '@trezor/theme';
+import { Box, Column, Icon, ProgressBar, Row, Text } from '@trezor/components';
+import { spacings } from '@trezor/theme';
 import { TranslationKey } from '@suite-common/intl-types';
 import { FirmwareOperationStatus, useFirmwareInstallation } from '@suite-common/firmware';
 
 import { Translation } from '../suite';
 
-const Wrapper = styled.div`
-    display: flex;
-    border-radius: ${borders.radii.xs};
-    padding: ${spacingsPx.lg} ${spacingsPx.xl};
-    width: 100%;
-    font-size: ${variables.FONT_SIZE.SMALL};
-    color: ${({ theme }) => theme.legacy.TYPE_LIGHT_GREY};
-    align-items: center;
-
-    ${variables.SCREEN_QUERY.ABOVE_LAPTOP} {
-        &:last-child {
-            border-radius: 0 0 ${borders.radii.md} ${borders.radii.md};
-        }
-    }
-`;
-
-const Label = styled.div`
-    display: flex;
-    margin-right: ${spacingsPx.lg};
-    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
-`;
-
-// eslint-disable-next-line local-rules/no-override-ds-component
-const StyledProgressBar = styled(ProgressBar)`
-    display: flex;
-    margin-right: ${spacingsPx.lg};
-    border-radius: ${borders.radii.xxs};
-    background: ${({ theme }) => theme.legacy.STROKE_GREY_ALT};
-    flex: 1;
-
-    ${ProgressBar.Value} {
-        height: 3px;
-        position: relative;
-        border-radius: ${borders.radii.xxs};
-    }
-`;
 const Percentage = styled.div`
-    display: flex;
-    margin-left: ${spacingsPx.sm};
-    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
     font-variant-numeric: tabular-nums;
-    height: ${spacingsPx.xl};
+    width: 30px;
```

Comment:
It's intended not have fixed width, because otherwise progress bar is moving according to percentage value or checkmark